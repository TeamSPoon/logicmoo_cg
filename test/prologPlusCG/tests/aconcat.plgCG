test(X) :- concat("ss", "yy", X).
test2(X) :- concat("ss", X, "ssyz").
test1(X) :- concat(X, "syz", "xsyz").
testBig(X, Y, Z) :- 
  concat(X, "yy", "ssyy"),
  concat("xx", Y, "xxyyy"),
  concat("xx", "yyysssyyy", Z),
  not(testno1), not(testno2), not(testno3).
testno1 :- concat("x", "y", "zas").
testno2 :- concat(X, "y", "zas").
testno3 :- concat("y", X, "zas").


             
