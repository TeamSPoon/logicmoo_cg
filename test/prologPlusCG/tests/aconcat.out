{A="ssyy", Z="xxyyysssyyy", C="yz", Y="yyy", B="x", X="ss"}
